indico/modules/events/papers/templates/emails/base.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'emails/base.html' %}
+
+{% block header %}{% endblock %}
+
+{% block footer %}{% endblock %}
+
+{% block subject -%}
+    {% block subject_message %}Call for papers{% endblock %}
+    {%- block subject_name %}{% endblock %}
+{%- endblock %}
+
+{% block body %}
+    <div class="mail" style="width: 600px; font-family: Arial; border: 1px solid #dfdfdf; background-color: #fcfcfc; margin: auto auto 40px auto;">
```

Comment:
What's the point of setting classes when we can't use non-inline CSS rules?
Also, do we really want to specify only `Arial` and not at least some fallback font available on non-Windows systems?